src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -91,29 +91,24 @@ export default function ConfirmedAppointmentListItem({
 
   let header;
   let location;
-  let videoSummary;
   if (isAtlas) {
-    header = 'VA Video Connect';
+    header = 'VA Video Connect at an ATLAS location';
```

Comment:
I think this will result in all of this text displaying as upper case, and it looked like from the ticket that only VA VIDEO CONNECT would be upper case.

You may want to verify that with Ryan and Lenae, since it will probably be a little bit of work to implement.

Reply:
I updated the code to resolve your concern